addons/main/functions/fnc_doUGL.sqf

```diff
@@ -66,14 +66,15 @@ _unit = _units deleteAt _unit;
 // force
 doStop _unit;
 _unit setUnitPosWeak "MIDDLE";
-_unit setVariable [QGVAR(forceMove), true];
+_unit setVariable [QEGVAR(danger,forceMove), true];
 
 // variable
 _unit setVariable [QGVAR(currentTask), "Shoot UGL", GVAR(debug_functions)];
 _unit setVariable [QGVAR(currentTarget), objNull, GVAR(debug_functions)];
+
 // dummy ~ seems necessary to get the AI to shoot up! -nkenny
 private _flarePos = [_pos, (_unit getPos [80, getDir leader _unit]) vectorAdd [0, 0, 200]] select (_pos isEqualTo []);
-private _dummy = "Target_F" createVehicle _flarePos;
+private _dummy = "Skeet_Clay_F" createVehicle _flarePos;
```

Comment:
why do we change that? didn't we tested it with Target_f and confirmed it works?

Reply:
I'm not sure if something changed or I missed it in the path.  But "Target_F" now actually spawns a _shooting target_ with a full 3d model on the given location.  The skeet clay was chosen because it is smaller and less obtrusive. (Especially in a dark night sky). 

Unfortunately using game logics or "module_F" has not yielded any success. The doTarget command only really works when assigned actual objects. :/

Reply:
Target_f is as far as I know a nothing that can be targeted